isisd/isis_circuit.c

```diff
@@ -1618,6 +1618,8 @@ static int isis_ifp_down(struct interface *ifp)
 	struct isis_circuit *circuit = ifp->info;
 
 	if (circuit) {
+		isis_area_switchover_routes(circuit->area, AF_UNSPEC, NULL,
```

Comment:
hmm - what does `isis_area_switchover` do with `AF_UNSPEC` ?
(hint - not much...)